### PR TITLE
chore: trying to fix some flaky tests

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Not 100% sure all are fixed, but should help with some stability issues hopefully 🤞 

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]
Probably good, we'll know after a few test runs in the wild

**Comment by rebelchris** [comment]
Oh the irony 🥲

This flaky test also happens locally, but at this point not even sure how it works sometimes 🤣

**Comment by omBratteng** [comment]
image

File: packages/shared/src/components/Feed.spec.tsx [hunk 1]
```diff
@@ -698,23 +691,19 @@ it('should report broken link with comment', async () => {
...
-  submitBtn.click();
-
-  await waitFor(async () => {
-    await screen.findByRole('alert');
-    const feed = await screen.findByTestId('posts-feed');
 
-    return expect(feed).toHaveAttribute('aria-live', 'assertive');
+  await act(async () => {
+    const [menuBtn] = await screen.findAllByLabelText('Options');
+    menuBtn.click();
```
Comment (capJavert) [inline packages/shared/src/components/Feed.spec.tsx:697]: Would not the correct way be to utilize `fireEvent` which wraps with `act` automatically, we did this to fix some flaky tests before.

Comment (rebelchris) [reply packages/shared/src/components/Feed.spec.tsx:697]: Ah yeah good point, can adjust.
Either way think these will work, but yes maybe cleaner

**Comment by rebelchris** [comment]
@omBratteng // @capJavert I commented the 2 flaky tests (with TODO) 
Really don't know, and tried 3 different approaches.

Will maybe look at it some other time, for now let's comment so they will not block us too much.
Also refactored all clicks :)

**Review by omBratteng — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]

**Pull request merged**